src/main/python/__cvd_risk_bmi.py

```diff
@@ -91,17 +94,37 @@ def heart_age_by_cvd_bmi(cvd,age,sex):
     else:
         cvd = float(cvd)
     if (sex == 'Male'):
-        X = BETA_MEN[1]*np.log(22) - CONST_MEN
+        X = BETA_MEN[1]*np.log(22) - CONST_MEN + CONST_MEN_BALANCER
         exp = np.exp(X)
         ln_cvd = np.log(1-cvd)/np.log(S_MEN)
         result = np.exp(((np.log(ln_cvd/exp))*(1/BETA_MEN[0])))
         return np.round(result)
     else
-        X = BETA_WOMEN[1]*np.log(22) - CONST_WOMEN
+        X = BETA_WOMEN[1]*np.log(22) - CONST_WOMEN + CONST_WOMEN_BALANCER
         exp = np.exp(X)
         ln_cvd = np.log(1-cvd)/np.log(S_WOMEN)
         result = np.exp(((np.log(ln_cvd/exp))*(1/BETA_WOMEN[0])))
         return np.round(result)
 
+def validate_heart_age(cvd,age,sex):
+    #handle the condition wheere cvd is None
+    if cvd is None:
+        return 0
+    if str(cvd) == n_a:
+        return 0
+    else:
+       split_gender = sex.split(",")
+       return 1
+
+def add_heart_age_adjuster(heart_age):
+    heart_age_adjuster : int = "1"
+    
+    print("Using heart age adjuster..")
+    return heart_age + heart_age_adjuster
+
 #cvd = frs('Male',10,33,26.3,0,0,1)
-#print heart_age_by_cvd(cvd,32,'Male')
+printr(heart_age_by_cvd(cvd,32,'Male'))
+heart_age = heart_age_by_cvd(cvd,32,'Female')
+hear_age_post_adjustment = add_heart_age_adjuster(heart_age)
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/adhirpotdarbito/pranahealth/pull/4#issuecomment-2303888136>#348dd9</a><span> - 08/22/2024, 06:34 am</span>
<br>

<div><small>🟠</small> Medium importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The variable name 'hear_age_post_adjustment' contains a typo and doesn't follow Python naming conventions.
</div>
<div id="fix">
<b>Fix:</b> Correct the typo in the variable name and use snake_case naming convention.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -128 +128 @@
 -hear_age_post_adjustment = add_heart_age_adjuster(heart_age)
 +heart_age_post_adjustment = add_heart_age_adjuster(heart_age)
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged